docs/pages/access-controls/device-trust/device-management.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Manage Trusted Devices (Preview)
+description: Learn how to manage Trusted Devices
+---
+
+<Admonition type="warning" title="Preview Note: Supported Device Type">
+The device trust preview works on macOS devices. Support for other operating
+systems and access features is planned for upcoming Teleport versions.
+</Admonition>
+
+### Prerequisites
+
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+
+## Register a trusted device
+
+The `tctl` tool is used to manage the device inventory. A device admin is
+responsible for managing devices, adding new devices to the inventory and
+removing devices that are no longer in use.
+
+<Admonition type="tip" title="Self enrollment: v13.3.5+">
```

Comment:
Since these are v12 docs, should we just remove anything that's specific to v13 and up?

Reply:
I usually add tip to tease features in new version so that reader may be tempted to upgrade. Happy to remove if it gives wrong impression though.